common/src/main/resources/Operator.properties

```diff
@@ -270,6 +270,8 @@ WLSDO-0063=The ''spec.configuration.initializeDomainOnPV'' is specified but ther
   ''spec.serverPod.volumes'' for the specified ''spec.configuration.initializeDomainOnPV.persistentVolumeClaim'' ''{0}''.
 WLSDO-0064=The ''spec.configuration.initializeDomainOnPV'' is specified but there is no volume in \
   ''spec.serverPod.volumes'' for PersistentVolumeClaim.
+WLSDO-0065=The OPSS wallet password secret ''{0}'' is specified but the required ''walletPassword'' key is missing \
+  in the specified secret ''{1}''.
```

Comment:
"in the specified secret ''{1}''" --> should this be "in the specified secret ''{0}'' since there's only one parameter passed.

Reply:
Never mind. I misread the message.